diff --git a/node_modules/react-native-vector-icons/fonts.gradle b/node_modules/react-native-vector-icons/fonts.gradle
index ddcf377..657f3ce 100644
--- a/node_modules/react-native-vector-icons/fonts.gradle
+++ b/node_modules/react-native-vector-icons/fonts.gradle
@@ -31,6 +31,12 @@ afterEvaluate {
         generateAssetsTask.dependsOn(fontCopyTask)
 
         def lintVitalAnalyzeTask = tasks.findByName("lintVitalAnalyze${targetName}")
-        lintVitalAnalyzeTask?.dependsOn(fontCopyTask)
+        if (lintVitalAnalyzeTask == null) {
+            lintAnalyzeDebug.dependsOn(fontCopyTask)
+            println "lintVitalAnalyzeTask lintAnalyzeDebug depends called "
+        } else {
+            lintVitalAnalyzeTask?.dependsOn(fontCopyTask)
+            println "lintVitalAnalyzeTask ${lintVitalAnalyzeTask} depends called "
+        }
     }
 }
