diff --git a/node_modules/react-native-svg/RNSVG.podspec b/node_modules/react-native-svg/RNSVG.podspec
index e28c609..f1dea05 100644
--- a/node_modules/react-native-svg/RNSVG.podspec
+++ b/node_modules/react-native-svg/RNSVG.podspec
@@ -15,10 +15,9 @@ Pod::Spec.new do |s|
   s.source_files    = 'apple/**/*.{h,m,mm}'
   s.ios.exclude_files = '**/*.macos.{h,m,mm}'
   s.tvos.exclude_files = '**/*.macos.{h,m,mm}'
-  s.visionos.exclude_files = '**/*.macos.{h,m,mm}'
   s.osx.exclude_files = '**/*.ios.{h,m,mm}'
   s.requires_arc    = true
-  s.platforms         = { :osx => "10.14", :ios => "12.4", :tvos => "12.4", :visionos => "1.0" }
+  s.platforms         = { :osx => "10.14", :ios => "12.4", :tvos => "12.4" }
 
   if fabric_enabled
     install_modules_dependencies(s)
